js/content/community.js

```diff
@@ -596,14 +581,33 @@ let ProfileHomePageClass = (function(){
             }
         }
 
-        if (SyncedStorage.get("profile_permalink")) {
-            document.querySelector("#es_permalink").addEventListener("click", function(e) {
-                e.target.select();
-            });
-            document.querySelector("#es_permalink_copy").addEventListener("click", function(e) {
-                document.querySelector("#es_permalink").select();
-                document.execCommand('copy');
-            });
+        function copySteamId(e) {
+            let elem = e.target.closest(".es_copy");
+            if (!elem) { return }
+
+            let text = elem.dataset.copy;
+            Clipboard.set(text);
+
+            ExtensionLayer.runInPageContext(`function() {
+                window.idDialog.Dismiss();
+                ShowAlertDialog(g_rgProfileData.personaname + "'s SteamID", \`${Localization.str.copied}\`.replace("__text__", "${text}"));
+            }`)
+        }
+
+        function showSteamIdDialog() {
+            let steamId = new SteamId(SteamId.fromDOM());
+            let html =
+                `<div class="bb_h1">${Localization.str.click_to_copy}</div>
+                <p><a data-copy="${steamId.getSteamId2()}" class="es_copy">${steamId.getSteamId2()}</a></p>
+                <p><a data-copy="${steamId.getSteamId3()}" class="es_copy">${steamId.getSteamId3()}</a></p>
+                <p><a data-copy="${steamId.getSteamId64()}" class="es_copy">${steamId.getSteamId64()}</a></p>
+                <p><a data-copy="https://steamcommunity.com/profiles/${steamId.getSteamId64()}" class="es_copy">https://steamcommunity.com/profiles/${steamId.getSteamId64()}</a></p>`;
+
+
+            ExtensionLayer.runInPageContext(`function() {
+                HideMenu("profile_action_dropdown_link", "profile_action_dropdown");
+                window.idDialog = ShowAlertDialog(g_rgProfileData.personaname + "'s SteamID", \`${html}\`, "${Localization.str.close}");
```

Comment:
The title should be localized too

Reply:
What is there to be localized about SteamID?

Reply:
This part: "'s"

Reply:
Uh.. right